shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -0,0 +1,122 @@
+from distutils.dir_util import copy_tree
```

Comment:
that's a duplicate of https://github.com/shimming-toolbox/shimming-toolbox-py/pull/18

having several unmerged branches that share the same development makes it difficult to keep track and to review (we end up making the same comments on both PRs...). is this PR necessary? or can it be re-focused?

Reply:
It's just that I'm using the content in this functionality. I most say, I would probably put it with the other PR since it is closely related. It was a separate task so I made a separate PR, but I think it would be better to put it together with the other one.